python/test/test_serialization.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+
+import opendp.prelude as dp
+
+
+atom = dp.atom_domain(bounds=(0, 10))
+
+@pytest.mark.parametrize(
+    "_readable_name,dp_obj",
+    [
+        (str(obj), obj)
+        for obj in [
+            # Domains:
+            atom,
+            dp.categorical_domain(['A', 'B', 'C']),
+            dp.series_domain('A', atom),
+            dp.lazyframe_domain([dp.series_domain('A', atom)]),
+            dp.wild_expr_domain([]),
+            # Metrics:
+            dp.absolute_distance("int"),
+            dp.change_one_distance(),
+            dp.linf_distance("float", True),
+            dp.user_distance("user_distance"),
+            # Measures:
+            dp.m.max_divergence(),
+            dp.m.approximate(dp.m.max_divergence()),
+            dp.m.user_divergence("user_divergence"),
+            # Measurements:
+            dp.m.make_gaussian(atom, dp.absolute_distance(int), 1),
+            dp.m.then_gaussian(1),
+            # Compositions:
+            (dp.vector_domain(dp.atom_domain(T=int)), dp.symmetric_distance())
+            >> dp.t.then_clamp((0, 10))
+            >> dp.t.then_sum()
+            >> dp.m.then_laplace(scale=5.0),
```

Comment:
Should also test that serde works for amplification combinator. Given that this works for the chainer combinator, I am optimistic that this is already working as intended.

Reply:
added a test: worked out of the box.